include/svs/core/allocator.h

```diff
@@ -41,7 +41,16 @@
 #include <array>
 #include <fcntl.h>
 #include <filesystem>
+
+/* <linux/mman.h> provides some linux-specific flags like
+ * MAP_POPULATE, MAP_NORESERVE, MAP_HUGETLB.
+ * They can be used only with linux, and are not availiable
+ * for MacOS
+ */
```

Comment:
Just a nitpick for the sake of consistency: can we change the comments with `//` instead of `/*`?  This applies to other comments in the PR